eos/signal-pdf.cc

```diff
@@ -214,28 +214,28 @@ namespace eos
                     Options{ {"l", "mu"} },
                     &BToDstarLeptonNeutrino::differential_branching_ratio,
                     std::make_tuple(
-                        KinematicRange{ "s", 0.0, 10.68, BToDstarLeptonNeutrino::kinematics_description_s }
+                        KinematicRange{ "q2", 0.0, 10.68, BToDstarLeptonNeutrino::kinematics_description_q2 }
```

Comment:
Should'nt we use here the real phase-space boundaries for q² without approximation, i.e.
-  q^2_min = m_\mu^2
-  q^2_max = (M_B - M_D*)^2 

or is this handled correctly upon initialization when the flavor of the B-meson spectator and the lepton flavor are known?